test/integration/MongooseAdapterSpec.js

```diff
@@ -72,4 +86,18 @@ describe('MongooseAdapterIntegration', function () {
       .catch(err => done(err))
     ;
   });
+
+  /* eslint-disable no-underscore-dangle */
+  it('allows to pass mongo ObjectId as a default attribute', function (done) {
+    mongoUnavailable && this.skip();
+
+    let thread;
+    factory.create('thread')
+      .then(created => (thread = created))
+      .then(() => factory.create('email', { thread: thread._id }))
+      .then(email => expect(email.thread).to.be.equal(thread._id))
+      .then(() => done())
+      .catch(err => done(err));
```

Comment:
Ah sorry, haven't thought of that. I'm used to use `then(done, done.fail)` with Jasmine.